dev/Makefile

```diff
@@ -1,6 +1,48 @@
 dev:
 	docker-compose -f docker-compose.yml up
-.PHONY: dev
+
+# Build or rebuild docker containers.
+build-dev:
+ifeq (,$(wildcard /usr/bin/yarn))
+	cd ../web; \
+	sudo apt-get update && sudo apt-get install -y yarn || \
+	apt-get update && sudo apt-get install -y yarn || \
+	sudo pacman --needed --noconfirm -S yarn || \
+	pacman --needed --noconfirm -S yarn; \
+	yarn install && \
+	yarn build; \
+	cd ../dev
```

Comment:
maybe we can have a 'build-web-local' target as this seems unexpected for a docker based environment?